readme.md

```diff
@@ -1,5 +1,14 @@
-# OpenRCT2
-An open-source re-implementation of RollerCoaster Tycoon 2. A construction and management simulation video game that simulates amusement park management.
+
+
+<p align="center">
+  <a href="https://openrct2.io">
+    <img src="https://raw.githubusercontent.com/OpenRCT2/OpenRCT2/develop/resources/logo/icon_x128.png" width="128px" alt="OpenRCT2 Logo"/>
```

Comment:
`width="128px"` is invalid HTML. Either use `width="128"` (deprecated) or `style="width: 128px"`